index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portolio Setup</title>
+  <link rel="stylesheet" href="./css/styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600;700;800;900&display=swap"
+    rel="stylesheet">
+</head>
+
+<body>
+  <header class="toolbar">
+    <img src="./img/menu-icon.svg" alt="Hamburger menu icon" class="hamburger-icon">
```

Comment:
Nice job so far, your project looks great!👍. Let's just attend to this little issue. For semantic reasons, kindly wrap the hamburger bar in a `nav` tag😉.